src/TaskList.svelte

```diff
@@ -76,6 +77,13 @@ export let scrollingDuration = 2000;
     padding-bottom: 0;
   }
 
+  .nes-container.auto-layout {
+    height: 100%;
+    max-height: 100vh;
+    width: fit-content;
```

Comment:
I think the width can stay as it was (full width) only change the height settings 

then it'll only auto size in the height